domainlab/arg_parser.py

```diff
@@ -12,6 +12,24 @@
 from domainlab.models.args_vae import add_args2parser_vae
 from domainlab.utils.logger import Logger
 
+class ParseValuesOrKeyValuePairs(argparse.Action):
+    """Class used for arg parsing where values are provided in a key value format"""
+
+    def __call__(self, parser, namespace, values, option_string=None):
```

Comment:
we need to change this signature to the style below:  since values here must be str

def surface_area_of_cube(edge_length: float) -> str:

Reply:
Added that with my last commit. However, we don't always use this notation throughout the code, which makes it a bit inconsistent.